app/src/lib/status-parser.ts

```diff
@@ -23,6 +23,18 @@ export interface IStatusEntry {
   readonly oldPath?: string
 }
 
+export function isStatusHeader(
```

Comment:
No need for this function, we can do this inline to reduce the amount of ceremony code.

Reply:
this is a [type guard](https://basarat.gitbooks.io/typescript/docs/types/typeGuard.html#user-defined-type-guards) which i'm pretty sure can't be an inline function. i put it in this file because it felt right to keep that defined near the type it applied to. let me know if you think it should live somewhere else, but we can't remove it.

this kind of typecasting function is [necessary when `Array.filter`ing a list by type](https://github.com/Microsoft/TypeScript/pull/10027)